Windows/ppsspp.rc

```diff
@@ -442,7 +449,22 @@ BEGIN
         MENUITEM "Control Mapping...",                      ID_OPTIONS_CONTROLS
         MENUITEM "Language...",                             ID_OPTIONS_LANGUAGE
         MENUITEM SEPARATOR
-        MENUITEM "Stretch to Display",                      ID_OPTIONS_STRETCHDISPLAY
+		POPUP "Display Layout Options"
+		BEGIN
+			MENUITEM "Full stretch", ID_OPTIONS_STRETCH
+			MENUITEM "Partial stretch", ID_OPTIONS_PARTIAL_STRETCH
+			MENUITEM "Auto scaling", ID_OPTIONS_DISPLAY_AUTO

```

Comment:
I don't know if a layperson will understand the difference between these three options.

-[Unknown]